src/app/containers/RadioSchedule/utilities/evaluateScheduleData.js

```diff
@@ -17,3 +18,37 @@ export const isScheduleDataComplete = ({
 
   return schedules[latestProgramIndex - 2] && schedules[latestProgramIndex + 1];
 };
+
+export const getIsProgramValid = logError => program => {
```

Comment:
Similar function here - https://github.com/bbc/simorgh/blob/latest/src/app/routes/utils/processRecentEpisodes/index.jsx#L4

Linking for visibility; I am not suggesting a refactor to make it work for both Recent Eps and Radio Schedules